shared/src/vyxal/elements/Elements.scala

```diff
@@ -748,39 +748,14 @@ object Elements:
         RegisterHelpers.push(a)
         if summon[Context].settings.registerPeek then push(a)
       },
-    "¥" -> 
-      direct(0) {
-        push(RegisterHelpers.pop(peek=true))
-      },
-    "`" ->
-      direct(0) {
-        push(RegisterHelpers.pop())
-      },
-    "Þ^" ->
-      direct(0) {
-        RegisterHelpers.reverseRegister()
-      },
-    "Þ⍨" ->
-      direct(0) {
-        for x <- RegisterHelpers.register.reverse do
-          push(RegisterHelpers.pop())
-      },
-    "Þ¥" ->
-      direct(0) {
-        push(RegisterHelpers.pop(allVals = true))
-      },
-    "Þw" -> 
-      direct(0) {
-        push(RegisterHelpers.pop(allVals = true, peek=true))
-      },
-    "Þ`" ->
-      direct(0) {
-        push(RegisterHelpers.length)
-      },
-    "Þ_" ->
-      direct(0) {
-        RegisterHelpers.clear()
-      },
+    "¥" -> niladify(ctx ?=> RegisterHelpers.pop(peek=true)),
```

Comment:
Are you able to do this now that `niladify` doesn't have a second overload now, btw? Or is the compiler too dumb to understand?
```suggestion
    "¥" -> niladify { RegisterHelpers.pop(peek=true) },
```